packages/components/Checkbox/src/assets/checkmark_ios.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="9" height="8" viewBox="0 0 9 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M0.853553 4.14646C0.658291 3.9512 0.341709 3.9512 0.146447 4.14646C-0.0488155 4.34172 -0.0488155 4.6583 0.146447 4.85357L2.64645 7.35357C2.85435 7.56147 3.19589 7.54597 3.38411 7.3201L8.38411 1.3201C8.56089 1.10797 8.53223 0.792684 8.32009 0.615902C8.10795 0.43912 7.79267 0.467782 7.61589 0.67992L2.9664 6.25931L0.853553 4.14646Z" fill="currentColor"/>
+</svg>
```

Comment:
Where did you get these svgs from? Figma?

Reply:
Yep vNext Figmas